### DISCUSSION TITLE
真人验证躲不过

### DISCUSSION THREAD

**Discussion by qinguoluo** [Harry-zklcdc/go-proxy-bingai#54] [General]
每次真人验证过后只能管1天，1天以后就出现无效域提示，需要到官网去再次验证，如此循环往复。

版本是最新的v1.10.1，KievRPSSecAuth变量在vercel和网页前端两处都是填了的，就是绕不过去这个验证。我不知道该怎么办了。

**Comment by qinguoluo** [comment]
今天又出现了，烦人！

**Reply by Harry-zklcdc** [reply]
看看最新版，会不会有这个问题

**Reply by qinguoluo** [reply]
v1.11.2版本还是这样。

**Reply by Harry-zklcdc** [reply]
看看这个解决方案

[wiki/解决-无效域&人机验证](https://github.com/Harry-zklcdc/go-proxy-bingai/wiki/解决-无效域&人机验证)
